test/unit/controllers/question.test.js

```diff
@@ -0,0 +1,88 @@
+const { expect, sinon } = require('test/chai-sinon');
+const { getQuestion, setupQuestionController } = require('app/controllers/question');
+const { INTERNAL_SERVER_ERROR } = require('http-status-codes');
+const appInsights = require('app-insights');
+const express = require('express');
+
+describe('question.js', () => {
+  const next = sinon.stub();
+  const req = {}, res = {};
+
+  req.params = {
+    hearingId: '1',
+    questionId: '2'
+  };
+
+  res.render = sinon.stub();
+
+  describe('getQuestion', () => {
+    // eslint-disable-next-line init-declarations
+    let questionService;
+
+    beforeEach(() => {
+      questionService = null;
+      sinon.stub(appInsights, 'trackException');
+    });
+
+    afterEach(() => {
+      appInsights.trackException.restore();
+    });
+
+    it('should call render with the template and question header', () => {
+      const questionHeading = 'What is the meaning of life?';
+      questionService = () => (
+        new Promise(resolve => {
+          resolve({
+            question_header_text: questionHeading
+          });
+        })
+      );
+      const getQuestionMiddleware = getQuestion(questionService);
+      return getQuestionMiddleware(req, res, next)
+        .then(() => {
+          expect(res.render).to.have.been.calledWith('question.html', {
+            question: {
+              header: questionHeading
+            }
+          });
+        });
```

Comment:
Have you used `async` and `await` much before? It can make this much more readable, but it's not a deal-breaker.